src/main/java/com/example/petbuddybackend/scheduled/RatingScheduled.java

```diff
@@ -15,12 +15,16 @@ public class RatingScheduled {
     private final RatingRepository ratingRepository;
     private final CaretakerRepository caretakerRepository;
 
-    @Scheduled(cron = "0 */30 * * * *")
+    /**
+     * Refreshing every 20 seconds for testing purpose, need to be changed to every 30 minutes for production
+     */
+    @Scheduled(cron = "0/20 * * * * *")
```

Comment:
Tylko dopytam, testing purposes czyli zostaje w repo?

Reply:
W sensie jak ma zostać to ok, ale pytam na zasadzie czy to tylko na testy do pull requesta jest czy na testy ogólnie

Reply:
narazie bym zostawil bo na przyszlosc zeby nie czekac 30 minut zeby przetestowac, bo kazdy odpala dockera i testuje od razu